### PR TITLE
Git support: git merge flow

### PR DESCRIPTION
**Pull request by lirantal**

### CODE REVIEW

**Comment by lirantal** [comment]
@Bonczidai want to review? :)

**Comment by Bonczidai** [comment]
:wink:

File: content/io/cloudslang/git/git_merge.sl [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# This flow merges a git branch to another branch
+#
+#   Inputs:
+#       - host - hostname or IP address
+#       - port - optional - port number for running the command
+#       - username - username to connect as
+#       - password - optional - password of user
+#       - git_repository_localdir - the target directory where a git repository exists and git_branch should be checked out to
+#                                 - Default: /tmp/repo.git
+#       - git_merge_branch - specify the branch to merge to
```
Comment (Bonczidai) [inline content/io/cloudslang/git/git_merge.sl]: specify the branch to merge **from**, right?

File: test/io/cloudslang/git/test_git_flow.sl [hunk 2]
```diff
@@ -48,11 +50,42 @@ flow:
...
 
+    - fetch_git_branch:
+        do:
+          git_fetch:
+            - host
+            - port
+            - username
+            - password
+            - git_fetch_remote
+            - git_branch
```
Comment (Bonczidai) [inline test/io/cloudslang/git/test_git_flow.sl]: `get_fetch` does not have `get_branch` input

**Comment by Bonczidai** [comment]
hey @lirantal,

added my small comments, please handle them and also merge from master (so the `git_fetch` changes on this branch will not overwrite the changes on master)

File: content/io/cloudslang/git/git_merge.sl (hunk 1 above)
Comment (lirantal) [inline content/io/cloudslang/git/git_merge.sl]: right, I'll update

**Comment by lirantal** [comment]
@Bonczidai updated.

**Comment by Bonczidai** [comment]
:+1:

**Comment by lirantal** [comment]
:)

**Pull request merged**